src/services/userAuthorization.js

```diff
@@ -22,7 +22,12 @@ function isActionAllowed(
 // Index CRUDL
 
 export const canSearchIndex = () => {
-  return isActionAllowed(store.state.user, 'index', 'list')
+  const indexListRight = store.state.user.rights.filter(rights =>
```

Comment:
From the look of it, it seems you're replacing a faulty `isActionAllowed` function with some custom code.

Why isn't `isActionAllowed` returning `allowed` on `index:list`, as it should be?

Reply:
The function `isActionAllowed` does not have sense for `index:list` and `collection:list` because these actions does not take in account the index and collection restricted in the user profile.